src/content/guides/hot-module-replacement.md

```diff
@@ -69,7 +69,7 @@ T> If you took the route of using `webpack-dev-middleware` instead of `webpack-d
   };
 ```
 
-T> You can use the CLI to modify the [webpack-dev-server](https://github.com/webpack/webpack-dev-server) configuration with the following command: `webpack serve --hot=only`.
+T> You can use the CLI to modify the [webpack-dev-server](https://github.com/webpack/webpack-dev-server) configuration with the following command: `webpack serve --hot-only`.
```

Comment:
`webpack serve --hot=only` should be now valid with the `webpack-cli` latest version.

https://github.com/webpack/webpack-cli/pull/2444

Reply:
I can confirm it doesn't work with `webpack-dev-server` 3.11.2.

![image](https://user-images.githubusercontent.com/1091472/114293677-ac511b80-9aca-11eb-96b4-d12fcc14fb33.png)

Reply:
Thanks for confirmation